bodhi/server/bugs.py

```diff
@@ -175,9 +177,14 @@ def modified(self, bug_id):
             log.exception("Unable to alter bug #%d" % bug_id)
 
 
-if config.get('bugtracker') == 'bugzilla':
-    log.info('Using python-bugzilla')
-    bugtracker = Bugzilla()
-else:
-    log.info('Using the FakeBugTracker')
-    bugtracker = FakeBugTracker()
+def set_bugtracker():
+    """
+    Set the module-level bugtracker attribute to the correct bugtracker, based on the config.
+    """
+    global bugtracker
+    if config.get('bugtracker') == 'bugzilla':
+        log.info('Using python-bugzilla')
+        bugtracker = Bugzilla()
```

Comment:
@pypingou We could do that, but I don't believe there would be harm in replacing it. If that happened, the old one would get garbage collected and things would keep on working.